src/WebJobs.Script.WebHost/BreakingChangeAnalysis/ChangeAnalysisService.cs

```diff
@@ -137,7 +137,7 @@ public IEnumerable<AssemblyReport> LogBreakingChangeReport(CancellationToken can
             {
                 cancellationToken.ThrowIfCancellationRequested();
 
-                AssemblyReport report = generator.ProduceReport(new Uri(assembly.CodeBase).LocalPath);
+                AssemblyReport report = generator.ProduceReport(new Uri(assembly.Location).LocalPath);
```

Comment:
Filed https://github.com/Azure/azure-functions-host/issues/7324 to track.

Reply:
Location is risky here in some scenarios, but I wouldn't worry about this code path too much, as this is something I'll be removing.